exporters/otlp/otlplog/otlploggrpc/example_test.go

```diff
@@ -31,3 +37,64 @@ func Example() {
 	// From here, the provider can be used by instrumentation to collect
 	// telemetry.
 }
+
+func ExampleWithTLSCredentials() {
+	ctx := context.Background()
+	var grpcExpOpt []otlploggrpc.Option
+	// the trusted certificate to use when verifying a server's TLS credentials
+	caFile := os.Getenv("OTEL_EXPORTER_OTLP_CERTIFICATE")
+	// the filepath to the client certificate
+	clientCert := os.Getenv("OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE")
+	// the filepath to the client's private key
+	clientKey := os.Getenv("OTEL_EXPORTER_OTLP_CLIENT_KEY")
```

Comment:
Using these env vars is very misleading. 

https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc#pkg-overview

> The configuration can be overridden by [WithTLSCredentials](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc#WithTLSCredentials),